src/components/swap/SwapModalFooter.tsx

```diff
@@ -139,7 +140,7 @@ export default function SwapModalFooter({
             allowedSlippage,
             transactionDeadlineSecondsSinceEpoch,
             isAutoSlippage,
-            isAutoRouterApi: !clientSideRouter,
+            isAutoRouterApi: routerPreference === RouterPreference.AUTO,
```

Comment:
i think you'll also want `|| routerPreference === RouterPreference.API` here

Reply:
its kind of confusing but rn "auto router" refers to using our routing api 🤔

Reply:
Yes, you are correct. Overlooked it!